api/document/js/schema.ts

```diff
@@ -34,10 +34,22 @@ const schema = new Schema({
     policy: {
       content: 'block+',
     },
-    inline: {
-      content: 'text*',
+    paraText: {
+      content: 'inline*',
       toDOM: () => ['p', { class: 'node-paragraph-text' }, 0],
     },
+    inlineFootnote: {
+      attrs: {
+        emblem: {},
```

Comment:
Should we give this a default? May reduce the number of confusing errors down the line...